php/swoole/start.php

```diff
@@ -4,7 +4,7 @@
 use Swoole\Http\Request;
 use Swoole\Http\Response;
 
-$server = new Server('0.0.0.0', 3000);
+$server = new Server('0.0.0.0', 3000, SWOOLE_BASE);
```

Comment:
is there a default value for this param @sy-records ?

Reply:
SWOOLE_PROCESS